src/WooCommerce.php

```diff
@@ -81,6 +81,11 @@ public static function getProductDetails($product_id = 0, $primary_category = TR
       'stock' => (int) $product->get_stock_quantity(),
     ];
 
+    // Adds product GTIN retrieved from a custom meta field.
+    if ($gtin = get_post_meta($product_id, Plugin::GTIN_CUSTOM_FIELD_NAME, TRUE)) {
```

Comment:
Technically, it would be more appropriate for shop-analytics to invoke a filter hook that allows all other plugins to participate in adding or modifying further values.

However, since core-standards is expected to be always there (so not really a third-party plugin), we can leave that step for a future point in time.